kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/WorkflowTester.kt

```diff
@@ -58,22 +59,23 @@ import kotlin.coroutines.EmptyCoroutineContext
  *    - Send a new [InputT] to the root workflow.
  */
 class WorkflowTester<InputT, OutputT : Any, RenderingT> @TestOnly internal constructor(
+  private val scope: CoroutineScope,
   private val inputs: SendChannel<InputT>,
-  private val host: WorkflowHost<OutputT, RenderingT>,
-  private val context: CoroutineContext
+  private val renderingsAndSnapshotsFlow: Flow<RenderingAndSnapshot<RenderingT>>,
+  private val outputsFlow: Flow<OutputT>
 ) {
 
   private val renderings = Channel<RenderingT>(capacity = UNLIMITED)
   private val snapshots = Channel<Snapshot>(capacity = UNLIMITED)
   private val outputs = Channel<OutputT>(capacity = UNLIMITED)
 
-  internal fun start() {
+  internal fun subscribeToWorkflow() {
```

Comment:
Nit: should we start saying "collectFrom" instead of "subscribeTo"?

Reply:
Yes.

- [ ] s/subscribeToWorkflow/collectFromWorkflow/